README.md

```diff
@@ -1,2 +1,156 @@
+# P2P Money
+
+## Application Link :link:
+
+[check it out from here](https://herokuapp.com/)
+
+## Team :busts_in_silhouette:
+
+### Team Lead
+
+- Asmaa Thabet
+
+### Team members
+
+- Hassan Al-Najjar
+- Ahmed Salah
+- Mohammed Flaifel
+- Ali Dahdouh
+
+## Application description :information_source:
+
+This is an app to help people exchange money online. without a middleman like a bank or exchange agency, we connect people through our app, this application is a good place for make profit from money exchange. we provide them with a wallet to save and exchnage their money anytime.
+currently, we are in the beta stage, we will provide virtual money for users to test our app, and review users feedback.
+
+## Application Setup :mag:
+
+- clone this repo
+  `git clone https://github.com/GSG-G8/p2p-money.git`
+
+- run `npm i && cd client npm i && npm run build` to install all dependencies & dev-dependencies for the app and also build react on the server.
+
+- add config.env file that contains SECRET_KEY as Environmental Variable and PostgreSQL Url for your database.
+
+#### Environment Variables
+
+Environment variables are one of the ways that we use to keep our product safe. If you want to access our app locally you will need to add your own.
+
+First create a **config.env** file and add the following variables:
+
+```
+SECRET_KEY
+```
+
+#### Database Setup
+
+- you can set the **database url** in your **_config.env_** as follows (setting the values in square brackets to the values you defined in PostgreSQL url):
+
+`postgres://[username]:[password]@localhost:5432/[database]`
+
+- Add a these two variables in config.env and added database url that you created as shown above :
+
+```
+DEV_URI = postgres://[username]:[password]@localhost:5432/[database]  // database for development
+PRODUCT_URI = postgres://[username]:[password]@localhost:5432/[database] // database for production
+```
+
+- Now in the Terminal run this command to build the fake data on your database on both development and production databases
+
+```
+npm run build:database:dev
+npm run build:database:pro
+```
+
+#### Run the Project
+
+- In terminal access to project folder on your machine write: `npm start` to start the app in the browser or you can work with development, write `npm run dev`.
+
+## Main Problems :heavy_exclamation_mark:
+
+1- Currency difference between exchange shops.
+2- waste the time and efforts when I search for the best price.
+3- Torn and damaged money.
+
+## Our Solutions :bulb:
+
+1- Gain the best Price between different prices.
+2- sea the different exchange prices for each bank and the money exchange agencies.
+3- manage your cash flow by states and charts.
+4- increase user profits.
+5- exchange your money in a secured and trusted environment.
+6- make changing money a simple process.
+7- reduce time and efforts
+8- flexible timing for exchange
+9- provide history stats.
```

Comment:
Leave spaces after each point to be displayed every one in separated line not  besides each other.